### PR TITLE
Paul/FREYA 723

### PR DESCRIPTION
**Pull request by Ziip-dev**
Add plotly scripts for serology plots generation.

### CODE REVIEW

**Comment by Ziip-dev** [comment]
Added README.md and requirements.txt

**Comment by LianeHughes** [comment]
Perhaps also put ticks on the x-axis so that it is clear that not every date is shown in the text written.

**Comment by Ziip-dev** [comment]
Thanks for the corrections that went under my radar :)
I'm working on the ticks and the box (might be a screen resolution issue, I'll increase the margin)

**Comment by Ziip-dev** [comment]
Finally done with the refining of the graphs (see attached screenshot).
I tried to play with `nticks` to display markers on the x axis for the hidden weeks, but it adds a lot of clutter at the cost of readability, so I prefer it this way. Please feel free to tell me if that is something we want anyway.

If we are all good, I'll rewrite the history locally to get all the commits signed (here and [there](https://github.com/ScilifelabDataCentre/pathogens-portal/pull/1138)) before merging the PR.

localhost_1313_dashboards_serology-statistics_

**Comment by Ziip-dev** [comment]
I rebased the entire branch to tidy the commit history up and sign everything again.
On my side, everything is now up to date :)

File: serology/cumulative-serology-tests.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    / sum_positive_negative
+)
+
+# Format accordingly
+proportion_positive = f"{proportion_positive:.2f}%"
+
+
+# Add figure annotation
+fig.add_annotation(
+    text=f"<b>Sum total</b><br>{sum_total}<br><br><b>Proportion positive</b><br>{proportion_positive}",
```
Comment (LianeHughes) [inline serology/cumulative-serology-tests.py:97]:
```suggestion
    text=f"<b>Sum total</b><br>{sum_total}<br><br><b>Proportion<br>positive</b><br>{proportion_positive}",
```

Comment (LianeHughes) [reply serology/cumulative-serology-tests.py:97]: The box is too wide for my viewport. If I add a break here, then it works fine. I have updated the image on blobserver with a version that reflects this change. See if it looks good for you too!

Comment (Ziip-dev) [reply serology/cumulative-serology-tests.py]: Still? :thinking: Strange, but ok! Your change is probably a better solution than continuing to increase the margin then :)

**Comment by Ziip-dev** [comment]
Changed output to stdout :+1:

**Comment by senthil10** [comment]
@Ziip-dev Looks good, one final question about the `*.lock` files. My understanding is (I might be wrong here), that we only need the config `json/toml` file and the `lock` files are created by devbox/poetry in runtime (and used more like cache) ? 

If thats the case, can we remove them and also add them to `.gitignore` so they are not included in the repo.

**Review by senthil10 — APPROVED** [review]

**Pull request merged**